storage/src/adb/current/mod.rs

```diff
@@ -67,6 +68,9 @@ pub struct KeyValueProofInfo<K: Array, V: CodecFixed<Cfg = ()>, const N: usize>
     /// The location of the operation that assigned this value to the key.
     pub loc: Location,
 
+    /// The next key in the key space, for ordered keyspaces.
+    pub next_key: Option<K>,
```

Comment:
Feels odd to me to include a field in a proof that isn't used on a variant 🤔 (not sure what the remedy would be but feels like "debris" we may unexpectedly run into)

Reply:
The alternatives I considered were to add a OrderedKeyValueProofInfo (either enum or separate struct). But given the overlap this felt less intrusive. Happy to reconsider.